testapp/README.rst

```diff
@@ -30,11 +33,11 @@ Quick start
                                         ('taggit_autocomplete_modified.urls'))
 
 
-4. Run 'python manage.py syncdb' to create models for the new installed apps.
+5. Run 'python manage.py syncdb' to create models for the new installed apps.
 
-5. Start the development server and visit http://localhost:8000/exam/
+6. Start the development server and visit http://localhost:8000/exam/
 
-6. In exam app run code_sever command  as superuser as follows::
+7. In exam app run code_sever command  as superuser as follows::
 
        $ code_server
```

Comment:
Sir, if the app is installed in a python virtual environment then sudo code_server command will look for the command in the system bin folder and therefore will not work. So the user has to activate the vitual environment as superuser and then just run code_server, it will work.
sudo code_server will be fine, if there is no virtual environment.